SampleBase/CMakeLists.txt

```diff
@@ -190,7 +190,7 @@ elseif(PLATFORM_ANDROID)
     target_link_libraries(Diligent-SampleBase PRIVATE GLESv3 PUBLIC native_app_glue)
 elseif(PLATFORM_LINUX)
     find_package(X11 REQUIRED)
-    target_link_libraries(Diligent-SampleBase PRIVATE XCBKeySyms ${OPENGL_LIBRARY} X11::X11)
+    target_link_libraries(Diligent-SampleBase PRIVATE XCBKeySyms ${OPENGL_LIBRARY} ${X11_LIBRARIES})
 elseif(PLATFORM_MACOS OR PLATFORM_IOS)
```

Comment:
Just wondering, why `X11::X11` does not work - this is supposed to be the official way to link with X11 using CMake.

Reply:
I have no idea why it doesn't work, I just know it doesn't.